src/components/NftRewards/NftRewardsSection.tsx

```diff
@@ -75,40 +78,69 @@ export function NftRewardsSection() {
     setPayAmount,
     setPayInCurrency,
     setPayMetadata,
+    validatePayAmount,
   } = payProjectForm ?? {}
 
   const converter = useCurrencyConverter()
   const payAmountETH =
     payInCurrency === ETH ? payAmount : fromWad(converter.usdToWei(payAmount))
 
+  const tierIdsFromIndices = (indices: number[]) => {
+    if (!rewardTiers) return []
+    return tierIdsFromTierIndices({
+      rewardTiers,
+      indices,
+    })
+  }
+
+  const sumSelectedTiers = (newSelectedIndices: number[]) => {
+    return newSelectedIndices.reduce((subSum, index) => {
+      return subSum + (rewardTiers?.[index]?.contributionFloor ?? 0)
+    }, 0)
+  }
```

Comment:
**suggestion** if these dont rely on any internal state, we can move them outside of the component's scope.

Reply:
ah nevermind, they do.